dll/win32/mpr/mpr.spec

```diff
@@ -56,38 +58,51 @@
 @ stdcall WNetCancelConnectionA(str long)
 @ stdcall WNetCancelConnectionW(wstr long)
 @ stdcall WNetClearConnections(ptr)
-@ stdcall WNetCloseEnum(long)
+@ stdcall WNetCloseEnum(ptr)
 @ stdcall WNetConnectionDialog1A(ptr)
 @ stdcall WNetConnectionDialog1W(ptr)
 @ stdcall WNetConnectionDialog(long long)
+@ stub WNetConnectionDialog2 # (ptr long wstr long)
+@ stub WNetDirectoryNotifyA
+@ stub WNetDirectoryNotifyW # (ptr wstr long)
 @ stdcall WNetDisconnectDialog1A(ptr)
 @ stdcall WNetDisconnectDialog1W(ptr)
 @ stdcall WNetDisconnectDialog(long long)
+@ stub WNetDisconnectDialog2 # (ptr long wstr long)
 @ stdcall WNetEnumCachedPasswords(str long long ptr long)
-@ stdcall WNetEnumResourceA(long ptr ptr ptr)
-@ stdcall WNetEnumResourceW(long ptr ptr ptr)
+@ stdcall WNetEnumResourceA(ptr ptr ptr ptr)
+@ stdcall WNetEnumResourceW(ptr ptr ptr ptr)
```

Comment:
This is not the first time someone tells you that `long` is perfectly fine for a handle..

Reply:
Removed.

Reply:
PS: Although I still think being explicit/consistent is better.